modules/cluster-logging-systemd-scaling.adoc

```diff
@@ -75,32 +75,46 @@ For more information on systemd settings, see link:https://www.freedesktop.org/s
 $ export jrnl_cnf=$( cat /journald.conf | base64 -w0 )
 ----
 
-. Create a new MachineConfig for master or worker and add the `journal.conf` parameters:
+. The following example creates a MachineConfig to change  `journald.conf` parameters for worker. You can change it to master or make an additional MachineConfig for the master role:
```

Comment:
The step needs to include an explicit action. I think this would be a better change:

~~~
.  Create a new MachineConfig for master or worker and add the `journal.conf` parameters:
** To change the `journald.conf` parameters for workers, create the following MachineConfig:
+
[source,terminal}
----
Your new example
----

** To change the `journald.conf` parameters for control plane machines, create the following MachineConfig:
+
[source,terminal}
----
Whatever it is for masters
----
~~~

Reply:
@kalexand-rh 
I am not against presenting both files, however to have it consistent with other parts for the doc I just presented one of them.
E.g. checking https://docs.openshift.com/container-platform/4.5/installing/install_config/installing-customizing.html we present either master, or worker or both. I am happy to contribute to overall more doc consistency, however, I am unsure which one it would be. If you say presenting both should be best practise I am going forward and add master config as well

Reply:
@dmoessne, I think that if we're making it easier for the user to paste their way to success that providing both is better. (I'm not opposed to presenting one and explaining how to modify it, if you think that that's more appropriate, but the step has to contain an explicit action statement. You'd just have to say something like "Create a MachineConfig to change the `journald.conf` parameters for a machine type. You must apply the `journald.conf` parameters for each machine type. The following command creates a MachineConfig for worker machines:" And then add a note after the example that explains how to modify the MachineConfig if you want to change the control plane machines.)